### PR TITLE
Added Caylus | MultiPlayer Game

### PR DESCRIPTION
**Pull request by JanumalaAkhilendra**
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -797,6 +797,19 @@
 | ------: | -------: | ------: |
 |   1 - 4 |       15 | 30-120m |
 
+### [caylus](https://boardgamegeek.com/boardgame/18602/caylus)
```
Comment (edm00se) [inline readme.md:800]: Please correct the title to be cased accordingly.

```suggestion
### [Caylus](https://boardgamegeek.com/boardgame/18602/caylus)
```

File: readme.md [hunk 2]
```diff
@@ -797,6 +797,19 @@
...
+
+>The players embody master builders. By building the King's castle and developing the city around it, they earn prestige points and gain the King's favor. When the castle is finished, the player who has earned the most prestige wins the game. The expansion Caylus Expansion: The Jeweller was included in the 2nd Edition.
```
Comment (edm00se) [inline readme.md:802]: The linter action states what it is failing for and it’s the lack of space between the `>` and the start of the text. Both this line and the one y one need an update.

```suggestion
> The players embody master builders. By building the King's castle and developing the city around it, they earn prestige points and gain the King's favor. When the castle is finished, the player who has earned the most prestige wins the game. The expansion Caylus Expansion: The Jeweller was included in the 2nd Edition.
```

File: readme.md [hunk 3]
```diff
@@ -797,6 +797,19 @@
...
+
+>Each turn, players pay to place their workers in various buildings in the village. These buildings allow players to gather resources or money, or to build or upgrade buildings with those resources. Players can also use their resources to help build the castle itself, earning points and favors from the king, which provide larger bonuses. Building a building provides some immediate points, and potentially income throughout the game, since players receive bonuses when others use their buildings. The buildings chosen by the players have a heavy impact on the course of the game, since they determine the actions that will be available to all the players.
+
+![caylus game image](https://cf.geekdo-images.com/yC7nOSc1x5PT-oNnh6TEcQ__itemrep/img/jVFZ85jsjJHQUXlImoDptZcENP8=/fit-in/246x300/filters:strip_icc()/pic1638795.jpg)
+
+
+| Players | Min. Age |    Time |
+| ------: | -------: | ------: |
+|   1 - 4 |       12 | 60-150m |
```
Comment (edm00se) [inline readme.md:811]: The number of players reported by BGG doesn’t match, and they’re generally quite reliable.

```suggestion
|   2 - 5 |       12 | 60-150m |
```

**Review by edm00se — CHANGES_REQUESTED** [review]
Just a couple of formatting notes. Once those are update I’ll be happy to merge this.

**Comment by JanumalaAkhilendra** [comment]
All the changes which were told have been made. Thank you for your consideration.

**Review by edm00se — APPROVED** [review]
Thanks for contributing!

Refs #269

**Pull request merged**